packages/dev/core/src/Loading/sceneLoader.ts

```diff
@@ -582,8 +608,8 @@ function loadData(
         // For plugin factories, the plugin is instantiated on each SceneLoader operation. This makes options handling
         // much simpler as we can just pass the options to the factory, rather than passing options through to every possible
         // plugin call. Given this, options are only supported for plugins that provide a factory function.
-        if (isFactory(registeredPlugin.plugin)) {
-            const pluginFactory = registeredPlugin.plugin;
+        if (isFactory(registeredPlugin!.plugin)) {
+            const pluginFactory = registeredPlugin!.plugin;

```

Comment:
These non-null assertions should not be needed, correct?

Reply:
Typescript disagreed with that (I was like you ;))